src/parsec_service/interfaces_and_dataflow.md

```diff
@@ -14,7 +14,7 @@ diagram. The shaded areas below can also be used to cross-reference the data flo
 code structure**](source_code_structure.md) document, which can help with finding the relevant
 definitions and implementations.
 
-![Data Flow Diagram](diagrams/interfaces_and_dataflow.png)
+![Data Flow Diagram](diagrams/interfaces_and_dataflow.svg)
```

Comment:
Nice! Changed to SVG so it can scale properly?

Reply:
Yes!! You can zoom in infinitely and also copy/paste text inside boxes